examples/ci_test.rs

```diff
@@ -341,15 +344,11 @@ fn main() {
         // Graceful exit
         {
             let &(ref lock, ref cvar) = &*stop_flg;
-            *unwrap_result!(lock.lock()) = true;
+            *unwrap!(lock.lock()) = true;
             cvar.notify_one();
         }
-    } else {
-        if let Some(log_file) = args.flag_output {
-            unwrap_result!(maidsafe_utilities::log::init_to_file(false, log_file, true));
-        } else {
-            unwrap_result!(maidsafe_utilities::log::init(false));
-        }
+    } else if let Some(log_file) = args.flag_output {
+        unwrap!(maidsafe_utilities::log::init_to_file(false, log_file, true));
```

Comment:
This should probably be left as it is, just with `unwrap_result!` replaced by `unwrap!`.